tenable/io/v3/base/schema/explore/search.py

```diff
@@ -1,27 +1,42 @@
 '''
 Base Explore Search Schema
 '''
-from marshmallow import Schema
-from marshmallow import fields as marshm_fields
+from marshmallow import Schema, fields, post_dump, pre_load
 from marshmallow import validate as v
 
-from .filters import FilterSchema
+from tenable.io.v3.base.schema.explore.filters import FilterSchema
 
 
 class SortSchema(Schema):
     '''
     Schema for the sorting sub-object
     '''
-    property = marshm_fields.Str()
-    order = marshm_fields.Str(validates=v.OneOf(['asc', 'desc']))
+    property = fields.Str()
+    order = fields.Str(validate=v.OneOf(['asc', 'desc']))
+
+    @pre_load(pass_many=True)
+    def transform_data(self, data, **kwargs):
+        if isinstance(data, tuple) and len(data) == 2:
+            property = data[0]
+            order = data[1]
+            return dict(property=property, order=order)
+        return data
+
+    @post_dump(pass_many=True)
+    def transform_request_data(self, data, **kwargs):
+        if not self.context['is_sort_with_prop']:
+            return {
+                data['property']: data['order']
+            }
+        return data
 
 
 class SearchSchema(Schema):
     '''
     Schema supporting the search request
     '''
-    fields = marshm_fields.List(marshm_fields.Str())
-    filter = marshm_fields.Nested(FilterSchema)
-    limit = marshm_fields.Int()
-    next = marshm_fields.Int()
-    sort = marshm_fields.List(marshm_fields.Nested(SortSchema))
+    fields_ = fields.List(fields.Str(), allow_none=True, data_key='fields')
```

Comment:
This shouldn't be necessary.  While you do have 2 things with the same name, they do not conflict.

fields != SearchSchema.fields

Reply:
![image](https://user-images.githubusercontent.com/93529282/146133156-24d3af46-fc0a-471b-a5fc-9204274fbdf4.png)
@SteveMcGrath if we use the fields as it is, then it throws error because fields variable is being re-assigned with list instead of marshmallow fields.

Reply:
![image](https://user-images.githubusercontent.com/93529282/146133453-0e4221c9-3678-4c74-9ec8-4d3d5ad88225.png)

Reply:
![image](https://user-images.githubusercontent.com/93529282/146133535-4fa82414-c5b1-4308-ad91-bc310b149486.png)

Reply:
interesting, alright.  fields_ it is then.